config.yml

```diff
@@ -144,8 +144,14 @@ params:
   categories:
     - name: Core
       description: Systems we have complete control over
-    - name: Others
-      description: Systems we use, but have no direct control over
+    - name: External Upstream CIs
```

Comment:
I would maybe go with `External/Internal *systems*`, CI may not really fit to all of these